Formula/gpgme.rb

```diff
@@ -12,6 +12,8 @@ class Gpgme < Formula
     sha256 "4a4c3e90175b02ded974a8a385b6d9799bcc2873c5a1087c558f350eb103934d" => :yosemite
   end
 
+  patch :DATA
```

Comment:
Thanks! Please add the link and short description as an inline Ruby comment and let's wait for a response.